src/1A5830.c

```diff
@@ -7,6 +7,8 @@
 
 s32 D_802946E0[] = { 100, 100, 100, 110, 130, 130, 130, 130, 130, 130, 130, 130, 130, 130, 130 };
 
+f32 update_lerp_battle(s32 easing, f32 start, f32 end, s32 elapsed, s32 duration);
```

Comment:
Rather put this in functions.h

Reply:
Since it is only used in this file and only in the functions that come after it, I think it's best to put the declaration next to INCLUDE_ASM